pkgs/desktops/gnome-3/core/baobab/default.nix

```diff
@@ -1,30 +1,54 @@
-{ stdenv, gettext, fetchurl, vala, desktop-file-utils
-, meson, ninja, pkgconfig, gtk3, glib, libxml2
-, wrapGAppsHook, itstool, gnome3 }:
-
+{ stdenv, lib, gettext, fetchurl, vala, desktop-file-utils
+, meson , ninja , pkgconfig , gtk3 , glib , libxml2
+, wrapGAppsHook , itstool , gnome3
+, gnused
+}:
 let
   pname = "baobab";
   version = "3.34.0";
 in stdenv.mkDerivation rec {
+
   name = "${pname}-${version}";
 
-  src = fetchurl {
-    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${name}.tar.xz";
-    sha256 = "01w6m5ylyqs4wslpjn1hg6n6ynwh3rghksak0qs8r9m6dm3dkss6";
-  };
+  srcs = [ (fetchurl {
+      url = "mirror://gnome/sources/${pname}/${lib.versions.majorMinor version}/${name}.tar.xz";
+      sha256 = "01w6m5ylyqs4wslpjn1hg6n6ynwh3rghksak0qs8r9m6dm3dkss6";
+    })
+    ./baobab-icons.tar.gz
+  ];
+
+  sourceRoot = "${name}";
 
   nativeBuildInputs = [ meson ninja pkgconfig vala gettext itstool libxml2 desktop-file-utils wrapGAppsHook ];
-  buildInputs = [ gtk3 glib gnome3.adwaita-icon-theme ];
+  buildInputs = [ gnused gtk3 glib gnome3.adwaita-icon-theme ];
 
   doCheck = true;
 
+  postPatch = ''
+    cp -r ../icons .
+  '';
+
+  # Deletes traductions for Icon Entry in .desktop
+  # And deletes the NotShowIn=KDE thing.
+  postBuild = ''
+    sed -i "/^Icon\[.*\].*$/d" data/org.gnome.baobab.desktop
+    sed -i "/^NotShowIn=KDE;$/d" data/org.gnome.baobab.desktop
```

Comment:
Ugh, [apparently](https://gitlab.gnome.org/GNOME/baobab/-/commit/0f8617de8aff245378c6f95028fbc396c296b6ef) some people [complained](https://bugzilla.gnome.org/show_bug.cgi?id=431117) when they saw GNOME utilities in KDE :woman_shrugging:

Reply:
Should I therefore close this PR?

Reply:
I think the reasoning is ridiculous and we should remove [this line](https://gitlab.gnome.org/GNOME/baobab/-/blob/328795a2674a3db9e36937c7203b1c61af1a6db6/data/org.gnome.baobab.desktop.in#L15) upstream.

I do not see the other changes as necessary, though.

Reply:
But I really find this unreasonable. If they don't wan't to see it, they just have to uninstall it. They can even  patch it and make their own baobab derivation with the NotShowIn entry.

Reply:
Yeah, I agree. Could you send a merge request against https://gitlab.gnome.org/GNOME/baobab removing the line?